x-pack/plugins/infra/public/lib/adapters/framework/kibana_framework_adapter.ts

```diff
@@ -132,6 +133,11 @@ export class InfraKibanaFrameworkAdapter implements InfraFrameworkAdapter {
       this.timezone = Private(this.timezoneProvider)();
       this.kbnVersion = kbnVersion;
       this.dateFormat = config.get('dateFormat');
+      try {
+        this.darkMode = config.get('theme:darkMode');
+      } catch (e) {
+        this.darkMode = false;
+      }
```

Comment:
This can be removed once the global dark theme PR is merged. For now, it's necessary to work with master.

Reply:
FYI, you can always specify a default as the second argument

Reply:
Oh... nice! I didn't know that would suppress the exception when the key doesn't exist.